.env.example

```
@@ -4,6 +4,7 @@
 # Needed for server side API calls
 NEXT_PUBLIC_SITE_URL="http://localhost:3000"
 API_BASE_URL="http://localhost:3000/api"
+PUZZLE_API="https://chess-puzzles.p.rapidapi.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)